src/openzaak/import_data/serializers.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2024 Dimpact
+from django.utils.translation import gettext as _
+
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import extend_schema_field
+from rest_framework.fields import SerializerMethodField
+from rest_framework.serializers import HyperlinkedModelSerializer
+
+from openzaak.import_data.models import Import, ImportStatusChoices
+
+
+class ImportSerializer(HyperlinkedModelSerializer):
+    status = SerializerMethodField(source="get_status")
+
+    @extend_schema_field(OpenApiTypes.STR)
+    def get_status(self, instance):
```

Comment:
Could you please explain why SerializerMethodField is needed here?

Reply:
For some reason, without this change, the schema generator thinks `ImportStatus` conflicts with the existing `StatusEnum` and generates a new enum, see [diff](https://gist.github.com/SonnyBA/2325c1c25060f765d960cb202d2f6000).